site/docs/bazel-and-java.md

```diff
@@ -47,17 +51,34 @@ The resulting binaries are compatible with locally installed JVM in system
 libraries, which means the resulting binaries depend on what is installed on the
 machine.
 
+To configure the JVM used for execution and testing use `--java_runtime_version`
+flag. The default value is `local_jdk`.
+
+### Hermetic Testing and Compilation
+
 To create a hermetic compile, you can use command line flag
 `--java_runtime_version=remotejdk_11`. The code is compiled for, executed, and
 tested on the JVM downloaded from a remote repository. For more details, see
 [Java runtime version flag](user-manual.html#flag--java_runtime_version).
 
+### Configuring Compilation and Execution of Build tools in Java
+
 There is a second pair of JDK and JVM used to build and execute tools, which are
```

Comment:
You mean ijar,singlejar, jacoco? Only the default values are set. They can be overridden.

Reply:
I mean that a tool used in the build process like a Java tool in a genrule would use `--tool_java_runtime_version`, but don't Java-based tools in `java_toolchain` like JavaBuilder use `java_toolchain.java_runtime`, which is separate from `--tool_java_runtime_version`? Or are `java_toolchain.java_runtime` and `--tool_java_runtime_version` still connected.

Reply:
I get it now. Yes, java_toolchain's tools execute using `java_toolchain.java_runtime`. I added a comment about this in the internal review, next to `java_runtime` attribute. I don't want to call it out in the use case for 'build tools`, because probably most of the users won't be modifying `java_toolchain`'s tools.